cloudstack/python/tar/slipstream_cloudstack/CloudStackClientCloud.py

```diff
@@ -363,6 +373,11 @@ def _size_get_ram(self, vm_size):
     def _size_get_instance_type(self, vm_size):
         return vm_size.name
 
+    def _find_root_disk_volume(self, instance_id):
+        for v in self.volumes:
+            if v.extra.get('volume_type') == 'ROOT' and v.extra.get('instance_id') == instance_id:
```

Comment:
I would swap the predicates. The best workload is the one that is never executed ;)